uast/uast-java/src/org/jetbrains/uast/java/declarations/JavaUFile.kt

```diff
@@ -8,15 +8,15 @@ import org.jetbrains.uast.*
 import org.jetbrains.uast.java.internal.JavaUElementWithComments
 import java.util.*
 
-class JavaUFile(
+class JavaUFile internal constructor(
```

Comment:
Note that this is the only one I can't make it `internal`, hence at least `internal constructor`. There are some tests that check cached evaluation results, and those somehow need copied version of this `JavaUFile` to not alter caches I guess.